docs/rest-api.md

```diff
@@ -145,10 +145,10 @@ python3 scripts/rest_client.py --config rest_config.json <command> [optional par
 | `locks` | Displays currently locked pairs.
 | `delete_lock <lock_id>` | Deletes (disables) the lock by id.
 | `profit` | Display a summary of your profit/loss from close trades and some stats about your performance.
-| `forceexit <trade_id>` | Instantly exits the given trade  (Ignoring `minimum_roi`).
-| `forceexit all` | Instantly exits all open trades (Ignoring `minimum_roi`).
-| `forceenter <pair> [rate]` | Instantly enters the given pair. Rate is optional. (`forcebuy_enable` must be set to True)
-| `forceenter <pair> <side> [rate]` | Instantly longs or shorts the given pair. Rate is optional. (`forcebuy_enable` must be set to True)
+| `force_exit <trade_id>` | Instantly exits the given trade  (Ignoring `minimum_roi`).
+| `force_exit all` | Instantly exits all open trades (Ignoring `minimum_roi`).
+| `force_enter <pair> [rate]` | Instantly enters the given pair. Rate is optional. (`forcebuy_enable` must be set to True)
+| `force_enter <pair> <side> [rate]` | Instantly longs or shorts the given pair. Rate is optional. (`forcebuy_enable` must be set to True)
```

Comment:
Please revert the changes to rpc method naming consistently, not just in some places.